app/components/candidate_interface/becoming_a_teacher_review_component.html.erb

```diff
@@ -8,5 +8,18 @@
     ),
   ) %>
 <% else %>
-  <%= render(SummaryCardComponent.new(rows: becoming_a_teacher_form_rows, editable: @editable)) %>
+  <div class="app-summary-card govuk-!-margin-bottom-6">
+    <% if @editable %>
+      <div class="app-summary-card__header govuk-body">
+        <%= govuk_link_to 'Edit your answer', candidate_interface_edit_becoming_a_teacher_path(return_to_params) %>
+      </div>
+    <% end %>
+    <div class="app-summary-card__body">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-three-quarters-from-desktop">
+          <%= simple_format(@application_form.becoming_a_teacher, class: 'govuk-body') %>
+        </div>
+      </div>
+    </div>
+  </div>
```

Comment:
Is there a limitation of the `SummaryCardComponent` that you had to work around here or is it just a completely different layout? (Just curious).

Reply:
Yep this version of the summary card doesn’t actually contain a summary list - as in this instance there’s no need for a “Personal statement” label in the first column because there’s only 1 question summarised and that’s the h1 anyway.

I’m not sure if it’s better to extend the summary card component to allow this variant someone, or treat this as a one-off?

Reply:
:+1: Sounds like a one-off.